src/main/resources/com/google/api/codegen/js/sample.snip

```diff
@@ -10,6 +10,10 @@
          https://console.developers.google.com/apis/api/{@class.apiName}
       2. Get access keys for your application. See
          https://developers.google.com/api-client-library/javascript/start/start-js#get-access-keys-for-your-application
+      @if class.auth.instructionsUrl
+        3. For additional information on making an authenticated call, see
```

Comment:
On the wording, I tend to lean more towards @tcoffee-google's suggestion. To your objection, @saicheems, we're not saying that auth is the only thing on the page. What if we word it as "For more information on using auth with JavaScript..."? But any of these three wordings are OK by me (but if we keep the original, s/an authenticated call/authenticated calls/)

But a more fundamental question: what useful information is the link providing that is not in the sign-in part of the sample below? AFAICT, the only difference is that the link has slightly fancier HTML. I'm not convinced that adding the link clarifies anything for the user.